src/Routes/RosPage/RosPage.js

```diff
@@ -124,7 +130,46 @@ class RosPage extends React.Component {
         ).then((results) => results);
     }
 
+    updateStateFilter = (value) => {
+        this.setState({
+            stateFilterValue: value
+        });
+    }
+
+    onDeleteFilters = (e, filtersArr) => {
```

Comment:
There's also `clearAll` flag you might want to check it out.

Reply:
@karelhala thanks for sharing, if you can share the link to the same that will be really helpful!